crc/models/approval.py

```diff
@@ -62,12 +71,22 @@ def from_model(cls, model: ApprovalModel):
         if model.study:
             instance.title = model.study.title
 
-        # TODO: Use ldap lookup
         instance.approver = {}
-        instance.approver['uid'] = 'bgb22'
-        instance.approver['display_name'] = 'Billy Bob (bgb22)'
-        instance.approver['title'] = 'E42:He\'s a hoopy frood'
-        instance.approver['department'] = 'E0:EN-Eng Study of Parallel Universes'
+        try:
```

Comment:
I'd prefer business logic go in a service or in an api file, but not in the models.  The models sole responsibility should be to describe the data they hold, and maybe convert data from one model to the other, but never call out to external services or perform major actions.